markdown/chapters/meandeck/interaction.md

```diff
@@ -743,50 +742,6 @@ worst case scenario, could take your Oracle out of your hand leaving you unable
 to win. Just be mindful of when these cards could come up if building a PTT
 pile.
 
```

Comment:
This section is fine but F my hard work wasted Q.Q

Reply:
Ikr 😢 it's all on the history though forever remembered